evalml/data_checks/multicollinearity_data_check.py

```diff
@@ -0,0 +1,50 @@
+
+from evalml.data_checks import (
+    DataCheck,
+    DataCheckMessageCode,
+    DataCheckWarning
+)
+from evalml.utils.gen_utils import _convert_to_woodwork_structure
+
+
+class MulticollinearityDataCheck(DataCheck):
+    """Check if any set features are likely to be multicollinear."""
+
+    def __init__(self, threshold=1.0):
```

Comment:
Maybe we should change the default threshold? I think [ww uses](https://github.com/alteryx/woodwork/blob/main/woodwork/datatable.py#L1000) sklearns [normalized_mutal_information](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.normalized_mutual_info_score.html) which is scaled between (0, 1) so this only issues a warning if columns are perfectly correlated.

Reply:
Agreed! Let's lower this a bit. What was the PS team doing?

Reply:
@dsherry @freddyaboulton I'll poke around or ask, but my understanding was that they just display the matrix, and don't use a threshold to determine whether things are correlated. I'll lower the threshold to 0.9--kind of arbitrary, but hard to determine without more testing 🤔

Reply:
@angela97lin ah yeah that makes sense. Sure, seems fine to me.